rotkehlchen/data_handler.py

```diff
@@ -75,6 +75,7 @@
     'balance_save_frequency',
     'anonymized_logs',
     'include_gas_costs',
+    'date_display_format',
```

Comment:
Hmm more changes are needed in the backend. You need to be able to return a default display format from the DB even if there is no value in there (for all people who have not yet upgraded to the changes of this PR).

Check here for how to achieve this: 

https://github.com/rotkehlchenio/rotkehlchen/blob/master/rotkehlchen/db/dbhandler.py#L391-L433

And also change the tests here:

https://github.com/rotkehlchenio/rotkehlchen/blob/dbdee73c98bc55940069d911eeebb65ed9825f1a/rotkehlchen/tests/test_db.py#L143